frontend/src/components/header/Header.vue

```diff
@@ -40,6 +40,11 @@ const props = defineProps({
             </Button>
             <span class="header-title">
                 {{ props.title }}
+                <span class="env-label">
+                    <strong>
+                        {{ environmentLabel }}
+                    </strong>
+                </span>
```

Comment:
just curious why we change the header?

Reply:
The env label was not showing on mobile

Reply:
um.... then we could make it more clean, like the following, so you don't need an extra class to hide the extra environmentLabel

```
 <span class="header-title">
        {{ props.title }}
         <strong class="subtitle">{{ props.subtitle }}</strong>
         <strong>{{ environmentLabel }}</strong>
 </span>
```